advanced-creation.md

```diff
@@ -82,6 +82,16 @@ const defaults = {
 				502,
 				503,
 				504
+			],
+			errors: [
```

Comment:
I think we should name it `errorCodes`, since they're from `error.code`.

Reply:
Same in the other places you use "error".

Reply:
I agree.